assets/js/blocks/cart-checkout/mini-cart/style.scss

```diff
@@ -1,12 +1,19 @@
-.wc-block-mini-cart {
+.wc-block-mini-cart.wp-block-woocommerce-mini-cart {
+	background-color: transparent !important;
 	display: flex;
 	justify-content: flex-end;
+
+	&.transparent-button .wc-block-mini-cart__button {
+		background: transparent !important;
+	}
 }
 
 .wc-block-mini-cart__button {
 	align-items: center;
 	background: transparent;
 	border: none;
+	color: inherit;
+	cursor: pointer;
```

Comment:
For accessibility reasons, I think we should leave the default cursor. `pointer` is used for links, but `default` is used for buttons and this behaves more like a button than a link.